src/core/Form/Select/MultiSelect/MultiSelect/__snapshots__/MultiSelect.test.tsx.snap

```diff
@@ -872,6 +872,7 @@ exports[`has matching snapshot 1`] = `
 
 <div
   class="c0 fi-multiselect c1"
+  items="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
```

Comment:
This seems to be leaking into the DOM due to not being destructured from props in MultiSelect after the changes.

Reply:
See comment below